### DISCUSSION TITLE
xrepo install zlib tbox 失败

### DISCUSSION THREAD

**Discussion by CoderXueXue** [xmake-io/xmake#5417] [Q&A]
操作系统 
MacOs 

 xrepo --version
xRepo 2.9.4+20240730/xmake, A cross-platform C/C++ package manager based on Xmake.

问题：
xrepo install zlib tbox
error: package(zlib) not found!
error: execv(/Users/huchai/.local/bin/xmake require -j 18 --extra={system=false} zlib tbox) failed(255)

**Comment by star-hengxing** [comment]
参考一下 https://github.com/xmake-io/xmake/issues/5366
